blueocean-core-js/package.json

```diff
@@ -5,8 +5,7 @@
   "main": "dist/js/index.js",
   "scripts": {
     "gulp": "gulp",
-    "test": "gulp test",
-    "prepublish": "gulp test; gulp"
```

Comment:
It runs before a `npm publish`, and also any time you run `npm install` without any parameters within the project folder. Running before a publish is fine; but in the other case, I've found it to be a real hassle.